ci/owncloud/upload_to_owncloud.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+from os import path
+from getpass import getuser
+import netrc
+from webdav3.client import Client
```

Comment:
Here again: just `Client` is not very clear, it could be any client. `webdav3.client.Client` is much clearer.